wp-content/themes/oceanwp-child/assets/css/template-styles.css

```diff
@@ -172,8 +177,6 @@ hr{
     color:#275f87
 }
 
-
-
 .blog-sub-header{
```

Comment:
Can we go with `subheader` and not `sub-header` ?

Reply:
Done